charts/prometheus-blackbox-exporter/templates/podsecuritypolicy.yaml

```diff
@@ -3,7 +3,6 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
```

Comment:
can we rather remove this completly?
it's deprectaed sind k8s 1.25 anyway: https://kubernetes.io/docs/concepts/security/pod-security-policy/

Reply:
Are there any news on this? Blackbox exporter is now blocking the Upgrade to 1.25 and Google may sson start to autoupgrade. I have removed it to test. Reading https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/ this seems save to do, as unless you have namespace based restrictions there are no restrictions in place.